Number_types/include/CGAL/Interval_nt.h

```diff
@@ -945,7 +945,7 @@ namespace INTERN_INTERVAL_NT {
       return approx.pair();
     }
     else
-      return std::pair<double,double>(l,l);
+      return std::pair<double,double>((double)l,(double)l);
```

Comment:
Please use `double(l)` or `static_cast<double>(l)`.

The C-style cast is an abomination without any checks, that allows to cast even carrots into space ships.